SourceCpp/Setup.cpp

```diff
@@ -129,20 +128,14 @@ PeleC::variableSetUp()
   // Get options, set phys_bc
   read_params();
 
-  // Initialize the runtime parameters for any of the external code
-  init_extern();
-
-  // Initialize the network
-  init_network();
-
 #ifdef PELEC_USE_REACTIONS
   // Initialize the reactor
   if (do_react == 1) {
     init_reactor();
   }
 #endif
 
-  init_eos();
+  EOS::init();
```

Comment:
it will be safer to call EOS::init before init_reactor. It may not break anything now but things. in init_reactor may need the chemistry to be initialized before.